Create rule S6718(secrets): Stripe Keys should not be disclosed (#2943)
SonarSource/rspec: shared_content/secrets/impact/blockchain_data_exposure.adoc

==== Disclosure of blockchain data

The leaked key can be used to query APIs of blockchain services and access

> sensitive information stored in the service metadata. This may include user identities and other sensitive data. +

Reviewer: ```suggestion
sensitive information stored in the service metadata. This may include user
identities and other sensitive data. +
```

==== Disclosure of blockchain data

The leaked key can be used to query APIs of blockchain services and access

> sensitive information stored in the blockchain. This may include transaction

Reviewer: ```suggestion
sensitive information stored in the service metadata. This may include transaction
```

> sensitive information stored in the blockchain. This may include transaction
> details, user identities, or other sensitive data. +

Reviewer: ```suggestion
sensitive information stored in the service metadata. This may include user identities and other sensitive data. +
```
